pkg/issuer/venafi/client/fake/venafi.go

```diff
@@ -19,23 +19,28 @@ package fake
 import (
 	"time"
 
-	"github.com/Venafi/vcert/pkg/endpoint"
+	"github.com/jetstack/cert-manager/pkg/issuer/venafi/client/api"
 
-	internalvanafiapi "github.com/jetstack/cert-manager/pkg/internal/venafi/api"
+	"github.com/Venafi/vcert/pkg/endpoint"
 )
 
 type Venafi struct {
 	PingFn                  func() error
-	SignFn                  func([]byte, time.Duration, []internalvanafiapi.CustomField) ([]byte, error)
+	RequestCertificateFn    func(csrPEM []byte, duration time.Duration, customFields []api.CustomField) (string, error)
+	RetreiveCertificateFn   func(pickupID string, csrPEM []byte, duration time.Duration, customFields []api.CustomField) ([]byte, error)
```

Comment:
`RetrieveCertificateFn`

Reply:
although that might be a vcert issue.. 😅

Reply:
no that was me...